### ISSUE TITLE
NPE while processing module results

### ISSUE THREAD

**Issue by zostay** [zostay/CPAN-Sidekick#12]
This came through from BugSense. I have no idea what this line is or what's the cause at this point.

```
java.lang.NullPointerException
at com.qubling.sidekick.ui.module.ModuleViewFragment.onFinishedFetch(ModuleViewFragment.java:175)
at com.qubling.sidekick.search.Search$1.run(Search.java:41)
at android.os.Handler.handleCallback(Handler.java:615)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4745)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
at dalvik.system.NativeStart.main(Native Method)
```

**Comment by zostay** [comment]
Saw this popup again in a crash report today.

```
java.lang.NullPointerException
at com.qubling.sidekick.ui.module.ModuleViewFragment.onFinishedFetch(ModuleViewFragment.java:177)
at com.qubling.sidekick.search.Search$1.run(Search.java:41)
at android.os.Handler.handleCallback(Handler.java:615)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:137)
```

**Comment by zostay** [comment]
Hmmm, that line number appears to be bogus. Code inspection time...

**Comment by zostay** [comment]
From what I can see, it appears that `getActivity()` is returning `null`. If that's the case, we don't want to try to update the documentation because we have no UI controls to update yet. However, if this is happening prior to the view being fully created, it should be refreshed when the activity is attached. If it is happening after the fragment has been taken out of the activity, it is about to destroyed and we don't care anymore anyway.

At least that's my theory for now. I'll close this and open a new issue if it recurs somehow.

Fixed in 76ea9c1c24f297efbb07e23941787008f75ea516 and will be in the next release.
